controller/user_blackbox_test.go

```diff
@@ -390,7 +390,7 @@ func newGormTestBase(identity *account.Identity, user *account.User) *GormTestBa
 		UserRepository:     TestUserRepository{User: user}}
 }
 
-func (rest *TestUserREST) createRandomUser(fullname string) account.User {
+func (rest *TestUserREST) giveUser(fullname string) account.User {
```

Comment:
Let's not rename methods/functions in this PR if don't have to. It will help us to make sure that we just are moving code from one file to another.

Reply:
@alexeykazakov then we will have to create an issue and PR just to rename this.

Reply:
That's OK.